### ISSUE TITLE
A question about the learning rate of pre-trained VGG 

### ISSUE THREAD

**Issue by Sucran** [Xyuan13/MSRNet#7]
You had created an amazing and large model based on pre-trained VGG model, but you set the same learning rate in the pre-trained models and almost all the following layers in your training. However, the scale of saliency datasets are much smaller than ImageNet dataset, and the results may suffered from overfitting problem. So, how did you solved this problem in you training ?
